.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
I guess... instead of just "instructions" I was thinking that we could give them a _template_ to fill out with information.